.gitlab-ci.yml

```diff
@@ -93,19 +93,6 @@ trigger_pipeline:
 
 # Build jobs
 # Job with example runs.
-# cuda 9.2 and friends
```

Comment:
Yes, we no longer maintain the corresponding container, so updating CMake on there is really hard

Reply:
OK, that makes sense. So perhaps these changes could be extracted and merged before this PR. I'm mostly asking this because I'm not familiar enough with this matter to properly review them.